skipchain/api.go

```diff
@@ -165,3 +165,12 @@ func (c *Client) GetSingleBlock(roster *onet.Roster, id SkipBlockID) (reply *Ski
 		&GetSingleBlock{id}, reply)
 	return
 }
+
+// GetSingleBlockByIndex searches for a block with the given index following the genesis-block.
```

Comment:
So a block can be referenced by its number (increasing id ?) in the chain AND/OR by its hash ?